trio/_sync.py

```diff
@@ -319,7 +327,7 @@ def release_on_behalf_of(self, borrower):
         self._borrowers.remove(borrower)
         self._wake_waiters()
 
-    def statistics(self):
+    def statistics(self) -> _CapacityLimiterStatistics:
```

Comment:
same here, if it's hinted expose the class properly instead of this underscore nonsense.